nomad-gcp/examples/basic/main.tf

```diff
@@ -54,9 +94,11 @@ module "nomad" {
 
   # Autoscaling for Managed Instance Group
   autoscaling_mode  = "ON"
-  nomad_auto_scaler = false # If true, will generate a service account to be used by nomad-autoscaler. The is output in the file nomad-as-key.json
-  max_replicas      = 4     # Max and Min replica values should match the values intended to be used by nomad autoscaler in CircleCI Server
-  min_replicas      = 1
+  nomad_auto_scaler = var.nomad_auto_scaler # If true, will generate a service account to be used by nomad-autoscaler. The is output in the file nomad-as-key.json
+  max_replicas      = var.max_replicas      # Max and Min replica values should match the values intended to be used by nomad autoscaler in CircleCI Server
+  min_replicas      = var.min_replicas
+  enable_workload_identity = var.enable_workload_identity     # If using GCP work identities rather than static keys in CircleCI Server
+
```

Comment:
Can we run a `terraform fmt` on this please?

Reply:
sure